### ISSUE TITLE
Compiler plugins stucks forever with openapi-annotation-processor v.5.1.4

### ISSUE THREAD

**Issue by YaroslavYakymenko** [javalin/javalin-openapi#106]
I have the following maven compiler config:

```
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-compiler-plugin</artifactId>
                <version>3.10.1</version>
                <configuration>
                    <annotationProcessorPaths>
                        <annotationProcessorPath>
                            <groupId>io.javalin.community.openapi</groupId>
                            <artifactId>openapi-annotation-processor</artifactId>
                            <version>5.1.4</version>
                        </annotationProcessorPath>
                    </annotationProcessorPaths>
                </configuration>
            </plugin>
```

The build stuck on compile phase.

It doesn't work either with JDK 11 and source/target 11 nor with JDK 17

However, it works fine in version 5.1.3 with JDK 17

**Comment by telearbeit** [comment]
In my case, it is a "from=File.class" in an OpenApiContent in an OpenApiResponse. Changing the File.class to any other type makes it work again.

@OpenApiResponse(status = "200", content = @OpenApiContent(type = "application/octet-stream", from = File.class))

**Comment by dzikoysk** [comment]
Thanks, looks like compiler API emits multiple `TypeMirror` instances on request for the same class, so we cannot use set to eliminate duplicates.

Btw I recommend to not use classes you don't control in OpenApi specification. `From` means that given body will be deserialized into this structure, but it's usually not truth for external types like e.g. `File`.

**Comment by dzikoysk** [comment]
It should be fixed in `5.1.4-2` :)
